docs/reference/commandline/service_rollback.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "service rollback"
+description: "The service rollback command description and usage"
+keywords: "service, rollback"
+---
+
+<!-- This file is maintained within the docker/cli Github
+     repository at https://github.com/docker/cli/. Make all
+     pull requests against that repo. If you see this file in
+     another repository, consider it read-only there, as it will
+     periodically be overwritten by the definitive file. Pull
+     requests which include edits to this file in other repositories
+     will be rejected.
+-->
+
+# service rollback
+
+```Markdown
+Usage:	docker service rollback SERVICE
+
+Rollback a service
+
+Options:
+  -d, --detach       Exit immediately instead of waiting for the service to converge (default true)
+      --help         Print usage
+  -q, --quiet        Suppress progress output
+```
+
+## Description
+
+Roll back a specified service to its previous version from the swarm. This command has to be run
+targeting a manager node.
+
+## Examples
+
+### Roll back to the previous version of a service
+
+This will revert the service to the configuration that was in place before the most recent docker service update command.
```

Comment:
"This" is a bit unclear, because here it doesn't have the introduction sentence that is present in the section it's copied from.

How about;

```Markdown
Use the `docker service rollback` command to roll back to the previous version
of a service. After executing this command, the service is reverted to the
configuration that was in place before the most recent `docker service update`
command.
````